modules/gdnative/godot/godot_vector2.cpp

```diff
@@ -28,267 +28,268 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 #include "godot_vector2.h"
+#include "core/variant.h"
 
-#include "math/math_2d.h"
+#include "core/math/math_2d.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 void _vector2_api_anchor() {}
 
-godot_vector2 GDAPI godot_vector2_new(const godot_real p_x, const godot_real p_y) {
-	godot_vector2 value;
-	Vector2 *v = (Vector2 *)&value;
-	v->x = p_x;
-	v->y = p_y;
-	return value;
+void GDAPI godot_vector2_new(godot_vector2 *r_dest, const godot_real p_x, const godot_real p_y) {
+
+	Vector2 *dest = (Vector2 *)r_dest;
+	memnew_placement(dest, Vector2(p_x, p_y));
```

Comment:
This doesn't need that.

Think of Vector2 and many other math classes like `int`s. You wouldn't do `memnew_placement()` for an integer